### PR TITLE
Doc free_memory and total_memory

### PR DESCRIPTION
**Pull request by kshyatt**

### CODE REVIEW

**Review by StefanKarpinski — APPROVED** [review]

File: base/sysinfo.jl [hunk 1]
```diff
@@ -234,7 +234,18 @@ function loadavg()
...
+    Sys.free_memory()
+
+Get the total free memory in RAM in kilobytes.
+"""
 free_memory() = ccall(:uv_get_free_memory, UInt64, ())
+
+"""
+    Sys.total_memory()
+
+Get the total memory in RAM (used and unused) in kilobytes.
```
Comment (KristofferC) [inline base/sysinfo.jl]: The `(used and unused)` comment made it feel like this returned a 2-tuple or something. Perhaps just

> Get the total memory of the RAM

or something?

Comment (kshyatt) [reply base/sysinfo.jl]: Maybe (including that which is currently used)? I want to make sure it's clear that it includes allocated memory too.

Comment (KristofferC) [reply base/sysinfo.jl]: :+1:

**Comment by kshyatt** [comment]
Failing `Pkg` test on OSX

**Comment by mauro3** [comment]
I think this is in bytes not kilobytes as on my 16GB ram system I get:
```
julia> Sys.total_memory()/1e9
16.62783488
```

**Pull request merged**